src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java

```diff
@@ -37,7 +37,7 @@ public ConstantJobCountScheduler(final JobSubmitter jobSubmitter, final Kubernet
     }
 
     private static boolean jobIsNot(Integer status) {
```

Comment:
This name doesn't quite match, seems that jobIsNotFailed (is that what 1 means?) + a constant might help with some confusion down the line?

Reply:
I use it for other statuses too, it's a funny API. The job has essentially several statuses (active, failed, etc) and then each of those is an Integer or null.

If it's 1 then it didn't fail. If it is null or non-1 it did, or maybe it hasn't gotten to that stage of its lifecycle yet.

Fair point on the constant.

Reply:
Here's how it gets called: 

```
                            if (jobIsNot(job.getStatus().getActive())) {
                                if (jobIsNot(job.getStatus().getFailed())) {
```